### ISSUE TITLE
Measure inference time per image

### ISSUE THREAD

**Issue by NanaNana22** [tensorflow/models#10595]
Hello,

I am training a pre-trained model (faster R-CNN with Resnet50) on Tesnoflow object detection API, and I need to **measure the inference time per image** (test set) using TensorFlow object detection API. Do you have any way to do that? please help me!

**Comment by pindinagesh** [comment]
@NanaNana22 

In order to measure inference time per image we can use **FLOPS** metric and for more information please refer [Measure inference](https://www.thinkautonomous.ai/blog/?p=deep-learning-optimization#flops_calc) and similar [issue](https://github.com/tensorflow/tensorflow/issues/32809). Thanks!

**Comment by NanaNana22** [comment]
@pindinagesh do you know how can I apply it on Tesnorflow Object detection API? the issue link you mention is applied on the hardcoded models.

**Comment by NanaNana22** [comment]
I see some developers using **time.time()** , is it correct?

**Comment by pindinagesh** [comment]
Hi @NanaNana22 

In order to expedite the trouble-shooting process, could you please provide the entire URL of the repository which you are using. Please provide more details on the issue reported here. Thanks!
